lib/models/repository-states/present.js

```diff
@@ -344,6 +355,7 @@ export default class Present extends State {
       async () => {
         try {
           await this.git().reset('soft', 'HEAD~');
+          addEvent('undoLastCommit', {package: 'github'});
```

Comment:
nit: for the counter names, we're using dash separation.  Could you make this `undo-last-commit` instead?